Silicon/ElkhartlakePkg/Include/Register/MrcOemPlatform.h

```diff
@@ -0,0 +1,47 @@
+/** @file
+  Sample implementation of SA_FUNCTION_CALLS member functions that are used by MRC,
+  and SPD read routines.
+
+@copyright
+
+ Copyright (c) 1999 - 2019, Intel Corporation. All rights reserved.<BR>
+ SPDX-License-Identifier: BSD-2-Clause-Patent
+
+@par Specification Reference:
+**/
+
+
+//#include <Library/PeiMemPolicyLib.h>
+//#include "MrcInterface.h"

```

Comment:
Remove these lines if since they are not used

Reply:
Removed, thanks for the review.